code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -17,6 +17,7 @@
 	var/list/dead_xenos = list() // xenos that are still assigned to this hive but are dead.
 	var/list/ssd_xenos
 	var/list/list/xenos_by_zlevel = list()
+	var/list/obj/structure/xeno/evotower/evotowers = list()
```

Comment:
autodoc, also could be a int if you're not doing a range check

Reply:
done